test/conformance/testdata/conformance.yaml

```diff
@@ -1445,6 +1445,34 @@
     watch event.
   release: v1.19
   file: test/e2e/network/service.go
+- testname: 'Network: EndpointSlice should create Endpoints and'
+  codename: '[sig-network] EndpointSlice should create Endpoints and EndpointSlices
+    for Pods matching a Service [Conformance]'
+  description: The EndpointSlice controller should create EndpointSlices for Pods
+    matching a Service.
+  release: v1.21
+  file: test/e2e/network/endpointslice.go
```

Comment:
I believe this content should be auto generated based on [structured comments like this](https://github.com/kubernetes/kubernetes/blob/master/test/e2e/network/ingress.go#L908-L919) above the tests and a run of `hack/update-conformance-yaml.sh`. There's a bit more info in the [conformance test readme](https://github.com/kubernetes/kubernetes/blob/master/test/conformance/README.md).

Reply:
Thanks! I was just looking for an example